docs/docsite/rst/dev_guide/developing_collections_distributing.rst

```diff
@@ -145,115 +166,116 @@ can set the following in your ``galaxy.yml`` file:
 
-   ansible-galaxy collection install git+https://github.com/org/repo.git,devel
+* Increment major (for example: x in `x.y.z`) version number for an incompatible API change.
+* Increment minor (for example: y in `x.y.z`) version number for new functionality in a backwards compatible manner (for example new modules/plugins, parameters, return values).
+* Increment patch (for example: z in `x.y.z`) version number for backwards compatible bug fixes.
 
-.. include:: ../shared_snippets/installing_collections_git_repo.txt
+Read the official `Semantic Versioning <https://semver.org/>`_ documentation for details and examples.
 
 .. _publish_collection:
 
-Publishing a collection
-=======================
+Publishing your collection
+==========================
 
-Once you have a namespace and an API token for each distribution server you want to use, and you have created and tested a collection tarball, you can distribute your collection by publishing the tarball to Ansible Galaxy, Red Hat Automation Hub, or a privately hosted Automation Hub instance. You can use either the ``ansible-galaxy collection publish`` command or the distribution server (Galaxy, Automation Hub) itself.
+The last step in distributing your collection is publishing the tarball to Ansible Galaxy, Red Hat Automation Hub, or a privately hosted Automation Hub instance. You can publish your collection in two ways:
 
-Each time you add features or make changes to your collection, you must create a new collection artifact and publish a new version of the collection. For details on versioning, see :ref:`collection_versions`.
+* from the command line using the ``ansible-galaxy collection publish`` command
+* from the website of the distribution server (Galaxy, Automation Hub) itself
 
 .. _upload_collection_ansible_galaxy:
-
 .. _publish_collection_galaxy_cmd:
 
-Publish a collection using ``ansible-galaxy``
+Publishing a collection from the command line
 ---------------------------------------------
 
-.. note::
-  By default, ``ansible-galaxy`` uses https://galaxy.ansible.com as the Galaxy server (as listed in the :file:`ansible.cfg` file under :ref:`galaxy_server`). If you are only publishing your collection to Ansible Galaxy, you do not need any further configuration. If you are using Red Hat Automation Hub or any other Galaxy server, see :ref:`Configuring the ansible-galaxy client <galaxy_server_config>`.
-
-To upload the collection artifact with the ``ansible-galaxy`` command:
+To upload the collection tarball from the command line using ``ansible-galaxy``:
 
 .. code-block:: bash
 
      ansible-galaxy collection publish path/to/my_namespace-my_collection-1.0.0.tar.gz
 
 .. note::
 
-	The above command assumes you have retrieved and stored your API token as part of a Galaxy server list. See :ref:`galaxy_get_token` for details.
+	The above command assumes you have retrieved and stored your API token in configuration. See :ref:`galaxy_specify_token` for details.
```

Comment:
Per the IBM style guide, try to avoid using above. This comment applies throughout the doc.

> above: previous, or can often be removed
Example:
Reenter the API key from the above field to confirm it.
Reenter the API key from the previous field to confirm it.

Reply:
Good catch. I've eliminated a few of the "above" references, but there are still quite a few.

Reply:
Fixed in 2b84951fc61422551f159ccf5df686beff5fb23e.